tests/utilities/test_xla_device_utils.py

```diff
@@ -37,12 +37,15 @@ def test_tpu_device_presence():
     assert xla_utils.XLADeviceUtils.tpu_device_exists() is True
 
 
-def test_result_returns_within_20_seconds():
+def test_result_returns_within_100_seconds():
     """Check that pl_multi_process returns within 10 seconds"""
 
+    # change timeout value during test
+    xla_utils.TPU_TIMEOUT_CONSTANT = 10
```

Comment:
Dont remember how, but would probably be safest if we patch this value rather than changing it globally. Will check later how to do it :)

Reply:
```python
@patch('xla_utils.TPU_TIMEOUT_CONSTANT', 10)
def test(...):
```
if I am not wrong.

Reply:
used `@patch('pytorch_lightning.utilities.xla_device_utils.TPU_TIMEOUT_CONSTANT', 10)`

Reply:
this is perfect!